src/main/java/com/google/graphgeckos/dashboard/fetchers/GithubFetcher.java

```diff
@@ -0,0 +1,23 @@
+package com.google.graphgeckos.dashboard.fetchers;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GithubFetcher {
```

Comment:
Should this be GitHub rather than Github?

Reply:
It should be:)